### ISSUE TITLE
Confirm current password when changing email address

### ISSUE THREAD

**Issue by fritzmg** [contao/contao#5196]
### Description

For an additional layer of protection it would be nice to be able to require to enter your current password again, if you try to change your login credentials or email address via the personal data module. This can be a feature that is enabled optionally. Thus you could split up the personal data changing process into two modules:

* One for being able to change email address, username & password (plus any additional sensitive data) where the additional confirmation is enabled).
* One for other personal data where confirmation of the current password is not required.

(_Note:_ by two modules I mean just creating two _Personal data_ modules with different settings.)

**Comment by leofeyer** [comment]
We already have the "change password" module for this. If you want your users to enter the current password, use this instead of the "personal data" module.

**Comment by fritzmg** [comment]
True, I forgot about that module. Then I don't think we need a new feature here.

**Comment by fritzmg** [comment]
Actually, the password changing module only allows you to change the password when entering the current password. There is currently no way to also require entering your current password if you try to change your email address.

**Comment by leofeyer** [comment]
Please adjust the issue title accordingly.
